discovery-infra/tests/base_test.py

```diff
@@ -116,6 +117,22 @@ def attach(node, disk_size):
         if modified_node is not None:
             modified_node.detach_all_test_disks()
 
+    @pytest.fixture()
+    def proxy_server(self):
```

Comment:
just to be sure. Commit is called "move  proxy_server fixture to base_test"  but i don't see any deleted fixture. What i am missing?

Reply:
it's moved over from the ds repo